pyne/alara.py

```diff
@@ -943,3 +1116,214 @@ def _find_phsrc_dc(idc, phtn_src_dc):
         # if idc doesn't match any string in phtn_src_dc list, raise an error.
         raise ValueError(
             'Decay time {0} not found in phtn_src file'.format(idc))
+
+
+def responses_output_zone(responses=None, wdr_file=None, alara_params=None):
+    """
+    This function returns a string representing the output zone of alara input
```

Comment:
Slightly better wording:

This function returns a string that is an output block for an alara input file, configured for zone resolution.